### ISSUE TITLE
Add chat to frontend

### ISSUE THREAD

**Issue by tikkat** [justarrived/just-match-web#298]
**Comment:**
Anna and Lova should be able to chat with users through the frontend. 
Please evaluate #322 before starting implementing since they both are building on the same design, maybe you want to modularize the code a little.

**Update:**
_Design suggestion_
![chat up](https://cloud.githubusercontent.com/assets/3510315/22986502/0fec4cd4-f3ac-11e6-8680-2a61c4a0f252.png)

Add the chat as an option in the side menu? :)

**Comment by buren** [comment]
Duplicate of https://github.com/justarrived/just-match-web/issues/258

**Comment by buren** [comment]
@tikkat 😻 ❤️

I agree! We should add `Chat` to the menu 👍

**Comment by bex1** [comment]
Is this still the design we want?
Should it be a separate page like this or available throughout the whole app?
@buren

**Comment by tikkat** [comment]
👍  @bex1 are you thinking about something similar to intercom? Might be a good idea as well

**Comment by buren** [comment]
@bex1 I'd say we could follow something that resembles comments. Would both be quick and easy to implement and we'd avoid introducing additional concepts to our users :)

Though the difference between comments and chat messages is that its not uncommon for chat messages to be fairly long (3-4 paragraphs), so we should make sure not to forget to test that. 

---

Later on Andrew will be allowed (:wink:) to do a more custom design.
